### PR TITLE
Adding scores

### PR DESCRIPTION
**Pull request by akhesaCaro**

### CODE REVIEW

File: src/GameBoard.hs [hunk 1]
```diff
@@ -89,28 +94,37 @@ data Paddle = Paddle
...
+    , ballLoc :: Position     -- ^ ball (x, y) location.
+    , ballVel :: Velocity     -- ^ ball (x, y) velocity
+    , ballDot :: Position     -- ^ velocity indicator
+    , bricks :: [Brick]       -- ^ bricks list
+    , paddle :: Paddle        -- ^ paddle
     } deriving Show
 
+
+mkBricks :: [Brick]
+mkBricks = map (flip Brick col) brickPos
```
Comment (gdeest) [inline src/GameBoard.hs:108]: Perhaps shouldn't this be called 'mkBricks', it's just a list of bricks.

File: src/Rendering.hs [hunk 2]
```diff
@@ -7,6 +7,11 @@ import GameBoard
 -- I want to use my own Vector.
 import Graphics.Gloss hiding (Vector)
 
+-- | Render score
+renderScore :: Score    -- ^ score to render
+            -> Picture  -- ^ picture of the score
+renderScore s = translate (-10) ((gameHeight / 2) - 50) $ scale 0.25 0.25 $ color blue $ Text (show s)
```
Comment (gdeest) [inline src/Rendering.hs:13]: This line is too long.

File: src/GameBoard.hs [hunk 3]
```diff
@@ -89,28 +94,37 @@ data Paddle = Paddle
...
+      where
+        grid = (,) <$> [-3..3] <*> [1..6]
+        brickPos = map (\(x, y) -> (x*brickStepX, y*brickStepY)) grid
+        col = yellow
+
+-- | Add one point to the score
```
Comment (gdeest) [inline src/GameBoard.hs:114]: "Add one point" doesn't seem accurate.

**Review by gdeest — APPROVED** [review]

**Comment by akhesaCaro** [comment]
Comments treated in the "mouse_controls" branch

**Pull request merged**